easybuild/easyconfigs/n/NEST/NEST-2.12.0-foss-2015b-Python-2.7.9.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'CMakeMake'
+
+name = 'NEST'
+version = '2.12.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.nest-simulator.org/'
+description = """NEST is a simulator for spiking neural network models
+ that focuses on the dynamics, size and structure of neural systems rather
+ than on the exact morphology of individual neurons."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/nest/nest-simulator/releases/download/v%(version)s/']
+sources = ['nest-%(version)s.tar.gz']
+checksums = ['bac578f38bb0621618ee9d5f2f1febfee60cddc000ff32e51a5f5470bb3df40d']
+
+configopts = '-DREADLINE_ROOT_DIR=$EBROOTLIBREADLINE -DCMAKE_INSTALL_LIBDIR=lib -Dwith-mpi=OFF'
```

Comment:
@jdonners Can you clarify the need for `-DCMAKE_INSTALL_LIBDIR=lib`, what happens if you don't specify that?

Reply:
I installed it both on Debian 7 and RedHat 7. On Debian it would install in 'lib', but on RedHat it would use lib64 by default. The sanity check then fails on RedHat 7.

Reply:
OK, that could also be handled by using `( 'lib/libnest.%s' % SHLIB_EXT,  'lib64/libnest.%s' % SHLIB_EXT)` in the sanity check (where `(..., ...)` specifies alternatives, one of those must be available), but this works too.

Thanks for testing on different OSs!